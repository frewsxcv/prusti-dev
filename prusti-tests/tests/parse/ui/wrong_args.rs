use prusti_contracts::*;

#[requires("too", "many", "args")]
fn too_many_args() {}

fn main() {}
