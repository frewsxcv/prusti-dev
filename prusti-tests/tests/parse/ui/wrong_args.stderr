error: unexpected token
 --> $DIR/wrong_args.rs:3:17
  |
3 | #[requires("too", "many", "args")]
  |                 ^

error: aborting due to previous error

